src/plugins/VariablePlugin.js

```diff
@@ -36,6 +36,7 @@ function VariablePlugin() {
    */
   const isEditable = ((value, code) => {
     const inVariable = value.inlines.size > 0 && value.inlines.every(node => node.type === 'variable');
```

Comment:
Same bug as in the ComputedPlugin with mixed inline selections?

Reply:
This one we still want using `.every`, because we only want `inVariable` to be true if the entire selection is a variable. we do not want to be able to highlight both non-editable text and a variable and still be able to edit the selection.